README.md

```diff
@@ -12,15 +12,7 @@ Welcome for any kind of contributions, such as open issues, fix bugs and improve
 
 ## Install
 
-**Note:** Currently we only support **Linux 64bit** environment by default, if you are not aware of what environment you are using, run the following command to check out.
-
-```python
->>> import platform
->>> print platform.system() # os
->>> print platform.architecture() # architecture
-```
-
-If you want to run py-common in your operating system, you need to use sdk-go-common tools to generate executables and replace them into directory `py-common/cryption/utils`. For more details please refer to [sdk-go-common](https://github.com/arxanchain/sdk-go-common/tree/master/crypto/tools/README.md)
+**Note:** If you want to run py-common in your operating system, you need to use **sdk-go-common** tools to generate executables and copy them into directory `py-common/cryption/utils`. For more details please refer to [sdk-go-common](https://github.com/arxanchain/sdk-go-common/tree/master/crypto/tools/README.md)
```

Comment:
Before using the py-common in your operation system, you need build crypto tools and copy them ....